src/Evolution/DiscontinuousGalerkin/Limiters/HwenoImpl.cpp

```diff
@@ -202,7 +202,7 @@ ConstrainedFitCache<VolumeDim>::ConstrainedFitCache(
   for (const auto& primary_dir : directions_with_neighbors) {
     if (directions_with_neighbors.size() == 1) {
       // With a single neighbor, there can be no neighbors to exclude.
-      std::vector<Direction<VolumeDim>> nothing_to_exclude{{}};
+      std::vector<Direction<VolumeDim>> nothing_to_exclude{};
```

Comment:
okay, this makes me worried we have other subtle bugs...   Should we add something like `Side::UnInitialized` and `Axis::UnInitialized` ?

Reply:
@kidder Good idea. I added these uninitialized enum values on `develop` and one bug popped out from running the unit tests (and one input file test) ... this one. Gold star to me! ⭐️

What do you think — should I clean open another PR to add `Side::Uninitialized` and `Axis::Uninitialize` ? I generally favor anything that helps to detect uninitialized variable problems...

Reply:
I'm in favor of having these in the main code base and making them be the first elements in the enum so that (hopefully) the compiler chooses them. I can't remember the rules on uninitialized enums, but from what I've seen it's always the first one.

Reply:
I like that; should probably do it for all enums...

Reply:
Ok, sounds good, I will work on this 👍